org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+rename-icon: gaphor
+command: gaphor
+finish-args:
+- "--socket=x11"
+- "--socket=fallback-x11"
+- "--share=ipc"
+- "--socket=wayland"
+- "--filesystem=xdg-run/dconf"
+- "--filesystem=~/.config/dconfig:ro"
+- "--talk-name=ca.desrt.dconf"
+- "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+cleanup:
+- "/include"
+- "/lib/pkgconfig"
+- "/share/pkgconfig"
+- "/share/aclocal"
+- "/man"
+- "/share/man"
+- "/share/gtk-doc"
+- "*.la"
+- "*.a"
+- "/lib/girepository-1.0"
+- "/share/doc"
+- "/share/gir-1.0"
+modules:
+- name: gaphor
+  buildsystem: simple
+  build-options:
+    build-args:
+    - "--share=network"
+    - "--filesystem=host"
+  build-commands:
+  - pip3 install --prefix=/app --disable-pip-version-check .
+  sources:
+  - type: git
+    tag: 1.0.0rc1
+    url: https://github.com/gaphor/gaphor
+    commit: 48c2bcff5ae32a6b6c4cae19d8d43634573741de
+  post-install:
+  - install -Dm644 flatpak/org.gaphor.Gaphor.appdata.xml /app/share/appdata/org.gaphor.Gaphor.appdata.xml
```

Comment:
Since you are the developer: Why not let `stetup.py` handle the installation:
https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/setup.py